### ISSUE TITLE
UI - Functional task

### ISSUE THREAD

**Issue by ZahidRavian** [asifjans/Inovedia#87]
Please mention all the pending functional issues here...

**Comment by ZarshaB** [comment]
**View Cars**
1>> It should not be editable.
![image](https://user-images.githubusercontent.com/35685146/35968574-1858ef22-0ce6-11e8-97d7-712963142350.png)

**Comment by ZarshaB** [comment]
**Car Detail**
1>> Filters should be filled with that specific car's information.

**Comment by ZarshaB** [comment]
**Home**
1>> Header Jerks.
2>>  Taking user to "Our Featured Cars" section but if screen is zoom out then it does not take to that section.

**Comment by ZarshaB** [comment]
**Register Page**

17>>  I have to select two times from DOB then errors disappears.
![image](https://user-images.githubusercontent.com/35685146/35797071-5e005ef6-0a80-11e8-84ea-060b6ca4d9ae.png)
18>> It should not accept Emirates Id less than the mentioned places. As now it is accepting if I only enter one digit. It should display error:
"Please enter valid Emirates Id."
![image](https://user-images.githubusercontent.com/35685146/35797080-68835428-0a80-11e8-8bb6-79f291990bb6.png)

**Comment by ZarshaB** [comment]
**Upload Ad** 
1) When I click on "Next" after giving title then on next page it displays error on all required fields. It should only display error if I click on "Next" button without filling required fields.
2) On clicking on Ad it redirects to blank page.
![image](https://user-images.githubusercontent.com/35685146/36138829-e9d808c4-10bc-11e8-87c3-0a17944f5022.png)

**Comment by ZarshaB** [comment]
**Order Summary**

1>> It should not allow me to buy tickets more than 750 for any car.
![image](https://user-images.githubusercontent.com/35685146/35852393-96a3df9a-0b4c-11e8-8690-3098095dd612.png)
2 >> If I update price then it reset Billing address field.

**Comment by asifjans** [comment]
7) Fixed

**Comment by ZarshaB** [comment]
**Welcome Email:**
It will contain add of 2-3 cars links which have sold highest number of tickets and are near raffle draw.
![image](https://user-images.githubusercontent.com/35685146/36145075-fd75fe1c-10d1-11e8-8fd0-b55c40f383ec.png)

**Comment by ZarshaB** [comment]
**Contact Us**
1>> Message field is a required field.
2>> Pop up message that will appear on submitting form is:
**Hi  (Name),
Thanks for reaching out. We’ll get back to you within 12 working hours.**
Instead of:
![image](https://user-images.githubusercontent.com/35685146/36145628-be8e506c-10d3-11e8-9804-81779a4e2a55.png)
3>> On submitting of form he'll get an email:
**Hi  (Name),
Thanks for reaching out. We’ll get back to you within 12 working hours.
In the meantime, why don’t you try your luck for your next ride here. [link to View Cars will be displayed as “here”]
Go get ’em tiger!!!**
4>> Please make City as a drop down as it is for "Register" page.
5>> Please make "Mobile Number" field same as it is for "Register" page with validation limit and +971 at the start.

**Comment by ZarshaB** [comment]
**My Profile**
1>> It should have upload picture option.

**Comment by ZarshaB** [comment]
**My Favourites**
1>> It should not repeat same ad in favourites.
![image](https://user-images.githubusercontent.com/35685146/36146601-052ec134-10d7-11e8-9e58-66904a176c0f.png)

**Comment by ZarshaB** [comment]
**Purchase History**
Ticket should have download to pdf link.

**Comment by ZarshaB** [comment]
**My Ads**
Status would be Ad Live, Car Sold and Ad Removed.
![image](https://user-images.githubusercontent.com/35685146/36153608-a8b8e316-10f0-11e8-9772-a49e09d51526.png)

**Comment by ZarshaB** [comment]
Domain should be "----@weladela.com" which is currently ".....@gmail.com".
![image](https://user-images.githubusercontent.com/35685146/36156400-bb1ec98c-10f8-11e8-887e-56498bdbda3d.png)

**Comment by ZahidRavian** [comment]
1>> It should not be editable.
2>> Once the 750 tickets has been sold"Buy Ticket" button should deactivate.
2>> Filters should be filled with that specific car's information.
3>> Header Jerks.
4>> Taking user to "Our Featured Cars" section but if screen is zoom out then it does not take to that section.
8>> On clicking on Ad it redirects to blank page.
10>> If I update price then it reset Billing address field.
11>> It is not updating complete reward points unless I logged out and then login.
12>> It shows total prize in minus if I use reward points more than total prize.
13>> If I use all reward points then it takes me to blank page on clicking on checkout button.

**Comment by ZarshaB** [comment]
39>> Price is appearing 0 on Home page.
![image](https://user-images.githubusercontent.com/35685146/36198186-b657f00a-1197-11e8-8d16-9bccd70edead.png)

**Comment by ZarshaB** [comment]
**Fixed:**
4 >> It is okay , 5,6,9,27,35
**Not Fixed:**
1,2,3

**Comment by ZarshaB** [comment]
40>> It should ignore points less than 100 AED.
41 >> Total reward points do not get reduce.
42>> If all reward points are used then on clicking on checkout it redirects to error page.

**Comment by ZarshaB** [comment]
**My Ads**
43>> Clicking on Ads redirecting to empty pages.
![image](https://user-images.githubusercontent.com/35685146/36207439-be71e5d2-11b7-11e8-82d8-335c02946463.png)

**Comment by ZarshaB** [comment]
44>> On clicking on Favourites, it should not take me to Car Detail page.

**Comment by ZarshaB** [comment]
45 >> Video should have play, pause etc. buttons.
![image](https://user-images.githubusercontent.com/35685146/36263076-1855723e-128b-11e8-8ea4-d5e89883b972.png)

**Comment by ZarshaB** [comment]
46>> It does not give me message if tickets have been sold and I can't buy more than 750. It does not allow me to update tickets.

**Comment by ZarshaB** [comment]
47>> It should remain on the same page if item is added into Favourites. From Home page, from pop up, from View Cars page or from Car detail page.

**Comment by ZarshaB** [comment]
48>> It should take here (Our Featured Cars" heading portion on login. 
![image](https://user-images.githubusercontent.com/35685146/36373078-ce56b052-1588-11e8-8d6a-94c7478871cc.png)

It is taking here on login.
![image](https://user-images.githubusercontent.com/35685146/36373111-eac1955e-1588-11e8-8b02-bf8444f2961e.png)

**Comment by ZarshaB** [comment]
49>> If more than one item is added into the cart then on applying reward points it redirects to blank page.
![image](https://user-images.githubusercontent.com/35685146/36375146-06fd4888-1590-11e8-9aff-395cd8c6b22b.png)

**Comment by ZarshaB** [comment]
50>> Transfer status is not coming on Purchase History.

**Comment by ZarshaB** [comment]
51>> User is not able to buy free ticket.

**Comment by ZarshaB** [comment]
52>> It did not update my remaining points if I used all my Reward points.
![image](https://user-images.githubusercontent.com/35685146/36420612-70aa54a2-1657-11e8-9fef-8eb33dd27fb1.png)

**Comment by ZarshaB** [comment]
53>> On clicking on "Continue" of empty shopping cart it should redirect to "Our Featured Cars" of Home page. Also message and button should be aligned.
![image](https://user-images.githubusercontent.com/35685146/36430966-3192ea60-1678-11e8-8b7f-f48605db3e51.png)

**Comment by ZarshaB** [comment]
54>> If without login I click on Cart then after login it should redirect to that page even if it is empty.

**Comment by ZarshaB** [comment]
55>> There should be discount at the end which will display the discount the user has got on using Reward points on Order Summary page.

**Comment by ZarshaB** [comment]
56>> Please make this alignment from "y" for both Order summary and Place your Ad pages.
![image](https://user-images.githubusercontent.com/35685146/36431163-b2ec11ea-1678-11e8-9b95-3673a7114ee7.png)

**Comment by ZarshaB** [comment]
57>> Please make 10 cars to be Featured on "Our Featured Cars" section.

**Comment by ZarshaB** [comment]
58>> In case of Refund, Reward Point History status would be:
Cancelled due to Ticket Refund
59>> if the customer wins a luck draw, then all his points will be forfeited and his balance will be zero.

**Comment by ZarshaB** [comment]
61>> When Ad is removed please change it's color. Instead of green you can change it to red and for "Delete Request" can change to orange or any other color.
![image](https://user-images.githubusercontent.com/35685146/36432514-2eef4a7a-167c-11e8-8f1d-81367670c5b7.png)

62>> Please blur the deactivate button to differentiate from other buttons.
![image](https://user-images.githubusercontent.com/35685146/36432535-418eb71a-167c-11e8-9afb-8bf5aa18c6c0.png)

**Comment by ZarshaB** [comment]
63>> Error message does not disappears after checking the box. It should disappear as it does for "Place your Ad" page.
![image](https://user-images.githubusercontent.com/35685146/36433119-996a6ab4-167d-11e8-95fb-db3eb5cd8086.png)

**Comment by ZahidRavian** [comment]
**These issues are fixed and ready for testing**

53>> On clicking on "Continue" of empty shopping cart it should redirect to "Our Featured Cars" of Home page. Also message and button should be aligned.
57>> Please make 10 cars to be Featured on "Our Featured Cars" section.
58>> In case of Refund, Reward Point History status would be:
Cancelled due to Ticket Refund
59>> if the customer wins a luck draw, then all his points will be forfeited and his balance will be zero. (This is admin related issue)
61>> When Ad is removed please change it's color. Instead of green you can change it to red and for "Delete Request" can change to orange or any other color.
62>> Please blur the deactivate button to differentiate from other buttons.
63>> Error message does not disappears after checking the box. It should disappear as it does for "Place your Ad" page.

**Comment by ZahidRavian** [comment]
**Pending issues are**

13>> There can be instances where people will try to post fake ads of the cars that they do not own. This will result is waste of our efforts and money. In order to ascertain the ownership of the car and to ensure that only valid cars are listed on our website, we will have an additional field in "Upload Ad". It will be labeled as "Mulkiya/Ownership Document" and will have an upload feature. JPGs, JPEGs and PDF file formats will be supported and will be a mandatory field. It will be for our own record to ascertain the ownership of the car and will not be displayed once the ad goes live.

14>> Also note that Mulkiya is like a card which has front and backside (refer to attachment). Therefore, this tab may require two upload buttons having "Front" and "Back" heading.

15>> Other Ownership Documents are in form of a single page. So it will require uploading of one file only.

16 >> So this tab can have two buttons with check box options. One will be labeled as "Mulkiya" and other one will be "Other Ownership Document". If the customer checks the Mulkiya button, two Upload features labelled as "Front" and "Back" will appear and customer will upload accordingly. If the customer checks "Other Ownership Document" then only one upload feature will appear.

24>> If Reward points are lower than Ticket Price then Pop-up will be displayed with redeem points terms.

25>> On clicking on "Proceed to checkout" button from checkout page user is asked for confirmation of order.

26>> When I add anything in cart while I'm logged in and without checkout I log out then again logged in then item should be remained in the cart. (Low Priority)

34>> It should have upload picture option.
38>> Domain should be "----@weladela.com" which is currently ".....@gmail.com".
46>> It does not give me message if tickets have been sold and I can't buy more than 750. It does not allow me to update tickets.
48>> It should take here ("Our Featured Cars" ) heading portion on login.
49>> If more than one item is added into the cart then on applying reward points it redirects to blank page.
50>> Transfer status is not coming on Purchase History.
51>> In case of Free ticket on clicking on "Checkout" user should be redirected to Purchase History page and should get the ticket.
52>> It did not update my remaining points if I used all my Reward points.
54>> If without login I click on Cart then after login it should redirect to that page even if it is empty.
55>> There should be discount at the end which will display the discount the user has got on using Reward points on Order Summary page.
60>> In case of Ticket Transfer , then for Purchase history:
If the Ticket Type is Transferred, then Date will be of transfer date, Price will be 0 and payment method will be N/A.

**Comment by ZarshaB** [comment]
**Not Fixed**
53>> Text and Button's alignment is fixed but not redirecting to "Our Featured Cars" of Home page.
63 >> Not Fixed.

**Comment by ZarshaB** [comment]
64>> It should display error on all required fields as user clicks on "Checkout" button without filling form.
![image](https://user-images.githubusercontent.com/35685146/36473498-7b55a04a-1716-11e8-97a7-c820b89b80a2.png)

**Comment by ZarshaB** [comment]
65>> It should be 6 days, 23 hours , 59 minutes and 60 seconds as I activate ad. Whereas it displays time with hours remaining of present day.
![image](https://user-images.githubusercontent.com/35685146/36478197-b7891632-1725-11e8-87e7-8e9efda90033.png)

**Comment by ZarshaB** [comment]
66>> It's button disappears on uploading image.
![image](https://user-images.githubusercontent.com/35685146/36482976-abd1d6a6-1736-11e8-85b0-157d0cc5605b.png)

**Comment by ZahidRavian** [comment]
**These issues are fixed and ready for testing**

25>> On clicking on "Proceed to checkout" button from checkout page user is asked for confirmation of order.
46>> It does not give me message if tickets have been sold and I can't buy more than 750. It does not allow me to update tickets.
49>> If more than one item is added into the cart then on applying reward points it redirects to blank page.
51>> In case of Free ticket on clicking on "Checkout" user should be redirected to Purchase History page and should get the ticket.
52>> It did not update my remaining points if I used all my Reward points.
53>> On clicking on "Continue" of empty shopping cart it should redirect to "Our Featured Cars" of Home page. Also message and button should be aligned.
54>> If without login I click on Cart then after login it should redirect to that page even if it is empty.

55>> There should be discount at the end which will display the discount the user has got on using Reward points on Order Summary page.
63>> Error message does not disappears after checking the box. It should disappear as it does for "Place your Ad" page.

**Comment by ZahidRavian** [comment]
**Pending Issues are**
 
13>> There can be instances where people will try to post fake ads of the cars that they do not own. This will result is waste of our efforts and money. In order to ascertain the ownership of the car and to ensure that only valid cars are listed on our website, we will have an additional field in "Upload Ad". It will be labeled as "Mulkiya/Ownership Document" and will have an upload feature. JPGs, JPEGs and PDF file formats will be supported and will be a mandatory field. It will be for our own record to ascertain the ownership of the car and will not be displayed once the ad goes live. **(new requirement)**
14>> Also note that Mulkiya is like a card which has front and backside (refer to attachment). Therefore, this tab may require two upload buttons having "Front" and "Back" heading. **(new requirement)**
15>> Other Ownership Documents are in form of a single page. So it will require uploading of one file only. **(new requirement)**
26>> When I add anything in cart while I'm logged in and without checkout I log out then again logged in then item should be remained in the cart. (Low Priority)
34>> It should have upload picture option.
38>> Domain should be "----@weladela.com" which is currently ".....@gmail.com".
50>> Transfer status is not coming on Purchase History.
58>> Status is not changed even after I have refund that.
60>> In case of Ticket Transfer , then for Purchase history:
If the Ticket Type is Transferred, then Date will be of transfer date, Price will be 0 and payment method will be N/A.
64>> It should display error on all required fields as user clicks on "Checkout" button without filling form. As it only displays error first on checkout and then after clicking on Checkout it displays error on Billing address.
65>> It should be 6 days, 23 hours , 59 minutes and 60 seconds as I activate ad. Whereas it displays time with hours remaining of present day.
66>> It's button disappears on uploading image.

**Comment by ZarshaB** [comment]
**Fixed** 
25,46,55,63,48,53,52,49

**Not Fixed**
54

**All Functional Fixed Issues are removed from this page.**

**Comment by ZarshaB** [comment]
67>> If I'm not login to Facebook then on clicking on "Continue with Facebook" it should redirect to Facebook login page instead of displaying this error.
![image](https://user-images.githubusercontent.com/35685146/36530616-5ff0e018-17dd-11e8-9a9d-2c9f96be4627.png)

**Comment by ZarshaB** [comment]
68>> Order Confirmation email is not displaying me discount even if I have given the discount.
![image](https://user-images.githubusercontent.com/35685146/36530666-85e03454-17dd-11e8-9f89-b0084ccff4e1.png)

**Comment by ZarshaB** [comment]
69>> It's placement is not correct and also not saving my profile picture.
![image](https://user-images.githubusercontent.com/35685146/36530782-d0e05812-17dd-11e8-808c-a9906283537a.png)

**Comment by ZarshaB** [comment]
70>> No email is being sent to Admin from live site.

**FYI**
71>> Contact Us and View Ad template is redirecting to demo site.

**Comment by ZarshaB** [comment]
70>> After login it is taking me here.
![image](https://user-images.githubusercontent.com/35685146/36541654-71ebea1a-1800-11e8-911c-9e2c8235eb81.png)

**Comment by asifjans** [comment]
(38) Fixed,

**Comment by Anum444** [comment]
**These issues are fixed and ready for testing**

54>> If without login I click on Cart then after login it should redirect to that page even if it is empty. **(Fixed)**
65>> It should be 6 days, 23 hours , 59 minutes and 60 seconds as I activate ad. Whereas it displays time with hours remaining of present day. **(Fixed)**
66>> It's button disappears on uploading image. **(FIXED)**
69>> It's placement is not correct (please check placement)  **(Fixed)**
70>> After login it is taking me here. **(Fixed)**

**Comment by ZarshaB** [comment]
**Fixed Issues:**
54,66,69,70,

**Not Fixed:**
65>> I uploaded an ad and showing me this time.
![image](https://user-images.githubusercontent.com/35685146/36548509-473032ec-1812-11e8-9df9-f33492abfb8f.png)

**Comment by ZahidRavian** [comment]
**Pending issues are**

58>> Status is not changed even after I have refund that.
64>> It should display error on all required fields as user clicks on "Checkout" button without filling form. As it only displays error first on checkout and then after clicking on Checkout it displays error on Billing address.
69>> It's placement is not correct and also not saving my profile picture.

**Comment by ZarshaB** [comment]
71>> Alignment Issue.
![image](https://user-images.githubusercontent.com/35685146/36598723-f109d626-18ce-11e8-9bd9-7500db88f5ba.png)

**Comment by ZarshaB** [comment]
72>> Broken image of logo is appearing on template.
![image](https://user-images.githubusercontent.com/35685146/36598738-04205d16-18cf-11e8-94ab-68a884fa0ed4.png)

**Comment by ZarshaB** [comment]
73>> This email template is having old domain of email.
![image](https://user-images.githubusercontent.com/35685146/36599040-e299214a-18cf-11e8-952d-22a1d4453179.png)

**Comment by ZarshaB** [comment]
60>> I purchased a ticket of Ad "Alfa Romeo 4c" and transferred that to "Lotus esprit s2". But In Purchase History it removed my history for "Alfa Romeo 4C" and in place of that it is showing new record of ad to which ticket is being transferred. 
![image](https://user-images.githubusercontent.com/35685146/36599673-fb756f78-18d1-11e8-8c5a-21e424f56660.png)

**Comment by ZarshaB** [comment]
61>> If without login user clicks on "Favorites" icon then it redirects to login page. On login page it displayed a message before. Now it is not displaying that message.

**Comment by ZarshaB** [comment]
63>> There should also be calendar for changing DOB. 
![image](https://user-images.githubusercontent.com/35685146/36663311-ff9eb6f8-1b02-11e8-8bd9-f5b3e6e59627.png)

**Comment by ZarshaB** [comment]
64>> If I click on "Favorites" then filter fills with car's info.
![image](https://user-images.githubusercontent.com/35685146/36664655-25f84bd0-1b07-11e8-8239-131bed2927a9.png)

**Comment by ZarshaB** [comment]
65>> If I buy something then after payment gateway I redirect to this page.
![image](https://user-images.githubusercontent.com/35685146/36664785-90b83d04-1b07-11e8-91e2-0adbf82f59b2.png)

**Comment by ZarshaB** [comment]
66>> On Refund it should change status in Reward points as "Cancelled due to refund.
![image](https://user-images.githubusercontent.com/35685146/36665610-dec83470-1b09-11e8-99f5-317d98bf9c84.png)

![image](https://user-images.githubusercontent.com/35685146/36665630-eb57cb92-1b09-11e8-9a6c-0724d620e673.png)

**Comment by ZarshaB** [comment]
67>> Make paging for "Reward Points" and "Purchase History" pages.
![image](https://user-images.githubusercontent.com/35685146/36678189-eef77892-1b31-11e8-8ea1-75132420ef74.png)

68>> Make headings for all pages of user's dashboard. As we have heading on "Place your Ad" portion in black.
![image](https://user-images.githubusercontent.com/35685146/36678333-4631a042-1b32-11e8-954c-dce71068bfd3.png)
![image](https://user-images.githubusercontent.com/35685146/36678346-4f87097a-1b32-11e8-82f2-05263ad53dfb.png)

**Comment by ZarshaB** [comment]
69>> Images are accepting files also. It should only accept images.
![image](https://user-images.githubusercontent.com/35685146/36730238-85ae1ee2-1be8-11e8-90c8-de21bfd51f70.png)

**Comment by asifjans** [comment]
67>> Make paging for "Reward Points" and "Purchase History" pages. 10 records should be there on each page. **(Fixed)**

**Comment by shayanr262** [comment]
**Not Fixed:**
63

**Comment by asifjans** [comment]
63>> There should also be calendar for changing DOB. (FIXED)

**Comment by asifjans** [comment]
69>> **(Fixed)**

**Comment by ZarshaB** [comment]
70>> If user is logged in then user's info should be auto populated in Contact Us form.
![image](https://user-images.githubusercontent.com/35685146/36789827-b6682966-1cb4-11e8-926f-1a4c90609827.png)

**Comment by ZarshaB** [comment]
71>> After payment gateway it is redirecting me to blank page.
![image](https://user-images.githubusercontent.com/35685146/36797058-e1adc356-1cc8-11e8-88f6-1f63a39e1490.png)

**Comment by ZarshaB** [comment]
69>> It allows me to upload any file in image section but on submitting ad it redirects me to blank page.

**Comment by ZarshaB** [comment]
**Not Fixed**
69,70

**Comment by asifjans** [comment]
(70) **Fixed**

**Comment by asifjans** [comment]
(69) Fixed

**Comment by ZarshaB** [comment]
74>> There is minus sign here.
![image](https://user-images.githubusercontent.com/35685146/36842154-8542548e-1d6c-11e8-9d25-46746f935021.png)

**Comment by ZarshaB** [comment]
75>> When I click on Emirates Id and then didn't enter anything, it displays error message that this information is required. It does not display this message for other fields. Please watch this video for reference.
![image](https://user-images.githubusercontent.com/35685146/36842415-72740af4-1d6d-11e8-8477-50f9284c9b1c.png)

Video Link = https://www.screencast.com/t/wANpc0iWR

**Comment by asifjans** [comment]
(73) **Fixed**

**Comment by ZarshaB** [comment]
75>> It either not accept these characters or should display error message "The information provided is not valid."
![image](https://user-images.githubusercontent.com/35685146/36848048-e5ef5e0c-1d81-11e8-8610-bfc0bda07da6.png)

**Comment by ZarshaB** [comment]
76>> It should not display this type of data.
![image](https://user-images.githubusercontent.com/35685146/36848298-a8540114-1d82-11e8-81ec-9de0b6710a7b.png)

**Comment by ZarshaB** [comment]
77>> When I add to Favorites from pop up then it redirects me to that Car's detail page.
Video Link = https://www.screencast.com/t/eP2qWwUv

**Comment by ZarshaB** [comment]
78>> When I do login with facebook then it takes me here. It should also take me to "Our Featured Cars" heading.
![image](https://user-images.githubusercontent.com/35685146/36900632-e1285f3a-1e45-11e8-93e6-e1145a3d1b68.png)
Video Link = https://www.screencast.com/t/WtuVELetSO

**Comment by ZarshaB** [comment]
79>> I'm having total reward points in negative.
Steps:
i) I had 440 total reward points.
ii) Due to refund I won and my total reward points were zero.
iii) I bought two tickets of 25 AED each. So my total reward points were 50.
iv) I refund a ticket. And that ticket was one of those whose reward points were zero because of winning of raffle draw. It's price was 125 AED.
v) And due to refund reward points were deleted. (50-125 = -75) 

![image](https://user-images.githubusercontent.com/35685146/36971804-d67006a6-208e-11e8-85e9-a7471e776557.png)

**Comment by ZarshaB** [comment]
80>> It should display car title, car image, price etc. so that user could know which ad/car is being transferred.
![image](https://user-images.githubusercontent.com/35685146/36977277-697c29da-20a2-11e8-9dd1-353bd81b3361.png)

**Comment by ZarshaB** [comment]
81>> If I register by using same email then it displays error of email already exists but also takes me to next page. It should not allow user to go to next page until all fileds are filled with correct data.
![image](https://user-images.githubusercontent.com/35685146/36978895-8ff2fdaa-20a7-11e8-8bac-7a5f00427898.png)

**Comment by ZarshaB** [comment]
82>> If car is sold then user will also get the email. User is not getting that email.
[Email Template to Successfull Seller.docx](https://github.com/asifjans/Inovedia/files/1781056/Email.Template.to.Successfull.Seller.docx)

**Comment by ZahidRavian** [comment]
This issue is fixed and ready for testing.
73>> The email domain is fixed for all other emails except for "Your Ad is live" template.

**Comment by ZarshaB** [comment]
83>> Profile picture does not save.

**Comment by ZarshaB** [comment]
84>> These fields should accept same data as they do on Register page. And also pattern of these fields should be same for example Emirates id does not have same pattern.
![image](https://user-images.githubusercontent.com/35685146/37021046-e7d39550-213f-11e8-9c8f-134c01e1f85b.png)

**Comment by ZarshaB** [comment]
85>> It is redirecting to demo site.
![image](https://user-images.githubusercontent.com/35685146/37021103-30f20140-2140-11e8-8d4c-48e895d6e955.png)

**Comment by ZarshaB** [comment]
86>> Redirecting to demo site.
![image](https://user-images.githubusercontent.com/35685146/37021377-24c414ac-2141-11e8-9424-fe5267d99104.png)

**Comment by ZarshaB** [comment]
87>> Please make everywhere total 600 tickets instead of 750.
![image](https://user-images.githubusercontent.com/35685146/37021715-56c5834a-2142-11e8-8edd-a66b597ec363.png)

**Comment by ZahidRavian** [comment]
These issues are fixed and ready for testing.

76>> It should not display this type of data.
78>> When I do login with Facebook then it takes me here. It should also take me to "Our Featured Cars" heading.
80>> It should display car title, car image, price etc. so that user could know which ad/car is being transferred.
82>> If car is sold then user will also get the email. User is not getting that email.
84>> These fields should accept same data as they do on Register page. And also pattern of these fields should be same for example Emirates id does not have same pattern.
85>> It is redirecting to demo site.
86>> Redirecting to demo site.
87>> Please make everywhere total 600 tickets instead of 750.

**Comment by ZarshaB** [comment]
89>> I entered correct DOB but it is showing me error.
![image](https://user-images.githubusercontent.com/35685146/37037188-90ccca72-2173-11e8-8f08-ebe5b841db65.png)

**Comment by ZarshaB** [comment]
90>> I uploaded ad and it's ticket sold was 20%. Although I didn't buy any ticket.

**Comment by ZarshaB** [comment]
91>> I uploaded ad. And after I approved that from Admin it's time was "6 days, 23 hours, 0 minutes and some seconds."
![image](https://user-images.githubusercontent.com/35685146/37082572-acd356b4-220e-11e8-9657-fedd32fce116.png)

**Comment by ZahidRavian** [comment]
These issues are fixed and ready for testing. 

81>> If I register by using same email then it displays error of email already exists but also takes me to next page. It should not allow user to go to next page until all fields are filled with correct data.
90>> I uploaded ad and it's ticket sold was 20%. Although I didn't buy any ticket.

**Comment by shayanr262** [comment]
81>> fixed
90>> fixed

**Comment by ZarshaB** [comment]
92>> It is accepting price 1 
![image](https://user-images.githubusercontent.com/35685146/37140232-d3f858ec-22d2-11e8-9af3-b621dc960f32.png)

**Comment by ZarshaB** [comment]
93>> It should remove ads those are removed from My Favorites.

**Comment by ZarshaB** [comment]
94>> There should also be "Pending" option so that Pending tickets could be searched on basis of status.
![image](https://user-images.githubusercontent.com/35685146/37145858-dcac122e-22e3-11e8-86c3-3d311b30553f.png)

**Comment by ZarshaB** [comment]
95>> Ticket's status should changed to "Pending" for "Deactivate" and "Sold" ads also in Purchase History.

**Comment by ZarshaB** [comment]
95>> Ad's title should not remain linked if Ad has status Pending, Deleted, Deactivated.

**Comment by ZarshaB** [comment]
96>> When ad is deactivated then on "My Ads" page, it's status becomes "N/A". It should be "Deactivated".

**Comment by ZarshaB** [comment]
97>> Facebook button is not working. Whenever I click on it, page is being refreshed only.

**Comment by ZarshaB** [comment]
Not Fixed>> 
91>> When I upload any ad it's time appears as "7 days, 9 hours, 9 minutes, and some seconds". When seconds becomes zero then correct time comes as "6 days, 23 hours, 59 minutes, 60 seconds".
![image](https://user-images.githubusercontent.com/35685146/37202630-e1c0c4a2-23ac-11e8-967b-db3003bdab9a.png)

**Comment by ZarshaB** [comment]
98>> My account has one active ad also but in message it is only telling me about one ticket.
![image](https://user-images.githubusercontent.com/35685146/37277142-a9014a82-2605-11e8-804e-2595e2176050.png)

**Comment by ZarshaB** [comment]
99>> The ad which whose time has expired should change status from Active.
![image](https://user-images.githubusercontent.com/35685146/37277554-dc2d9388-2606-11e8-8d24-5f4d7f23fe0a.png)

**Comment by ZarshaB** [comment]
100>> When I delete my Account from Admin and sign up with same email then it does not remove my reward points, my purchase history and my ads from that account.

**Comment by ZarshaB** [comment]
101>> It will also display timer.
![image](https://user-images.githubusercontent.com/35685146/37294802-76318f80-2638-11e8-8efe-d401af84fa20.png)

**Comment by ZarshaB** [comment]
102>> If I use reward points and then remove that ad and then again buy ticket of any other ad. Then on Order Summary page it has already subtracted the same reward points that I used for previous ad and deleted that ad.
![image](https://user-images.githubusercontent.com/35685146/37351586-3a68ca84-26fd-11e8-9a5f-de49f62388ac.png)

**Comment by ZarshaB** [comment]
103>> It is showing me negative reward points.
![image](https://user-images.githubusercontent.com/35685146/37351626-541e6f2e-26fd-11e8-8f02-f1e8d08f317b.png)

**Comment by ZarshaB** [comment]
104>> If I enter "e" here then it let me enter so many numbers 🐙 
![image](https://user-images.githubusercontent.com/35685146/37405089-9686ba20-27b5-11e8-896a-e09b4c3ba0c8.png)

**Comment by ZarshaB** [comment]
105>> On clicking on "watch" icon, image flicks and then go to original size.
Video Link = https://www.screencast.com/t/ex4UxGJQ

**Comment by ZarshaB** [comment]
106>> On clicking on "Facebook button" it is redirecting me to "Edit Profile" page.
![image](https://user-images.githubusercontent.com/35685146/37530287-01aac71e-295b-11e8-9cf5-80c10c3a4bfc.png)

**Comment by ZarshaB** [comment]
106>> If I login with facebook then if I don't update my profile then it allows me to access other pages and also I can purchase ticket. Only after purchase it redirects me to an error page. And also on clicking on "Reward Pints" page it displays error. 
![image](https://user-images.githubusercontent.com/35685146/37584038-ab9e4a86-2b75-11e8-80d9-5b59aba3b864.png)

**Comment by ZarshaB** [comment]
107>> When I move pages for "Reward Points" then for "Redeem History" both pages have same data whereas Redeem history is having only one page data. But moving on next pages for "Reward Points" also move "Redeem points".

![image](https://user-images.githubusercontent.com/35685146/37587177-6054f18c-2b80-11e8-84a2-eeadcd12a9cd.png)

**Comment by ZarshaB** [comment]
108>> I've deleted this ad from Admin. But in Purchase history, it is still appearing linked and on clicking on title it redirects to page and show status as "Sold out".

![image](https://user-images.githubusercontent.com/35685146/37587866-545e5a6a-2b82-11e8-9714-4e88c0eb1cf8.png)

**Comment by ZarshaB** [comment]
109>> It is giving me discount of 11 for 595 Reward points. Whereas it should not count reward points less than 100.
![image](https://user-images.githubusercontent.com/35685146/37645705-6f2e4a36-2c49-11e8-9329-c7ed57d720b0.png)

**Comment by ZarshaB** [comment]
110>> It is displaying me reward points earned as 105. Whereas I have paid AED 94 and I have earned 94 reward points.
![image](https://user-images.githubusercontent.com/35685146/37645830-c8c3240e-2c49-11e8-8d34-686b69148d73.png)
![image](https://user-images.githubusercontent.com/35685146/37645860-dab0fcb8-2c49-11e8-94ec-2d6a57d17fb9.png)

**Comment by ZarshaB** [comment]
**I'm seeing this on Demo. This issue has occurred after back button functionality is fixed.** 
111>> When I click on any ad in order to ad it to favorites, it's heart color does not change to orange but message is displayed that item is added. I have to refresh the page and then heart's color changes to filled or unfilled. 
Also When I click on "Favorites" from watch pop up then it displays so many pop ups at back end.

**Comment by ZahidRavian** [comment]
Pending issues are:
106 >> If I login with facebook then if I don't update my profile then it allows me to access other pages and also I can purchase ticket. Only after purchase it redirects me to an error page. And also on clicking on "Reward Pints" page it displays error.

107>> When I move pages for "Reward Points" then for "Redeem History" both pages have same data whereas Redeem history is having only one page data. But moving on next pages for "Reward Points" also move "Redeem points".

108>> I've deleted this ad from Admin. But in Purchase history, it is still appearing linked and on clicking on title it redirects to page and show status as "Sold out" **(Zahid)**

110>> It is displaying me reward points earned as 105. Whereas I have paid AED 94 and I have earned 94 reward points. (Need discussion)

**Comment by ZarshaB** [comment]
**Not Fixed:**
94

**Improvement:**
99>> Buttons are now deactivated. But Please chnage it's color to grey. Like we have "Apply Reward Points" button color when button is deactivated.

**Comment by asifjans** [comment]
106) Fixed

**Comment by asifjans** [comment]
107. When I move pages for "Reward Points" then for "Redeem History" both pages have same data whereas Redeem history is having only one page data. But moving on next pages for "Reward Points" also move "Redeem points".  **(Fixed)**

**Comment by ZarshaB** [comment]
94 is not fixed for me.

**Comment by ZarshaB** [comment]
Other issues are pending. I'll update after testing.

**Comment by ZarshaB** [comment]
Comments are added with remaining issues.

**Comment by asifjans** [comment]
110>> It is displaying me reward points earned as 105. Whereas I have paid AED 94 and I have earned 94 reward points. **(Can  you please try to reproduce this issue by created new account)**

**Comment by ZarshaB** [comment]
111>> It should not give discount for any points less than 100.

**Comment by ZarshaB** [comment]
112>> If I click on cross then favorites icon does not update. 
![image](https://user-images.githubusercontent.com/35685146/38361621-805c67ca-38e7-11e8-867e-2164f07b8e67.png)

**Comment by asifjans** [comment]
111 (Fixed, Please confirm)

**Comment by ZarshaB** [comment]
111>> It should not give discount for any points less than 100. (Not Fixed)

**Comment by ZarshaB** [comment]
112>> Latest does not appear on first page but appear at the last.
![image](https://user-images.githubusercontent.com/35685146/38499182-3055b470-3c20-11e8-9a8e-02a170e60ac3.png)
![image](https://user-images.githubusercontent.com/35685146/38499204-3cf901be-3c20-11e8-9050-7e87d9ed1f7b.png)
